browser/src/control/Control.JSDialog.js

```diff
@@ -77,7 +77,7 @@ L.Control.JSDialog = L.Control.extend({
 			if (this.dialogs[id].timeoutId)
 				clearTimeout(this.dialogs[id].timeoutId);
 
-			if (this.dialogs[id].isPopup)
+			if (this.dialogs[id].isPopUp)
```

Comment:
There is other occurrence of `isPopup` in the code, in general it was the original name of the property, but it was changed in the rework at some point: https://github.com/CollaboraOnline/online/commit/a2d666d53adcb23e60ed61328e8e3fe44fdc1af5

Can we rather change where it is set to `isPopUp` -> `isPopup` to be consistent with other occurrences?
Here: https://github.com/CollaboraOnline/online/blob/93825c688ccb0430bff1b5114996230a7615013c/browser/src/control/Control.JSDialog.js#L557C11-L557C19